src/control/lib/hardware/sysfs/provider.go

```diff
@@ -140,7 +122,8 @@ func (s *Provider) topologySubsystems() []string {
 	return netSubsystems
 }
 
-func (s *Provider) addPCIDevices(topo *hardware.Topology, subsystem string) error {
+func (s *Provider) addSubsystemDevices(topo *hardware.Topology, subsystem string) error {
+	netvscPaths := make([]string, 0)
```

Comment:
(very minor) nit: If you want to specify a slice without length or capacity and let append handle the underlying array and memory allocation for you I think it's more clear to use `var netvscPaths []string` rather than specifying a slice of explicitly zero length.

Reply:
- [ ] Remove useless slice initialization with make